src/task/RetryableTask.ts

```diff
@@ -48,7 +44,14 @@ export class RetryableTask extends WhenAllTask {
      * @param child
      *  The sub-task that just completed
      */
-    public trySetValue(child: TaskBase): void {
+    public trySetValue(child: TaskBase, executor?: TaskOrchestrationExecutor): void {
+        if (!executor) {
+            throw new Error(
+                "No executor passed to RetryableTask.trySetValue. " +
+                    "A TaskOrchestrationExecutor is required to schedule new tasks."
+            );
```

Comment:
Can we phrase this in such a way that it's clear to users that this is a framework-internal error instead of a user-generated error? Ideally we'd also recommend that they open an issue on github to report this. Just trying to practice giving actionable error messages.

Reply:
Sure, do you have a specific wording in mind?

Reply:
Just start with "framework internal exception: <what the error is>." and end with "Consider reporting this issue at <github url>"

Reply:
Done